examples/albert/arguments.py

```diff
@@ -45,10 +45,6 @@ class AveragerArguments:
     averaging_timeout: float = field(
         default=30.0, metadata={"help": "Give up on averaging step after this many seconds"}
     )
-    listen_on: str = field(
```

Comment:
Request: before merging, please also check that this argument is removed from tutorials/examples

Reply:
I've checked this, all remaining occurrences of `listen_on` are related to MoE.